app/controllers/book_requests_controller.rb

```diff
@@ -4,5 +4,28 @@ def new
   end
 
   def create
+    @book_request = current_user.book_requests.build(book_request_params)
+    if @book_request.save
+      insert_data
```

Comment:
Method's name should be describe its correct action

Reply:
fixed